modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -153,9 +161,11 @@ var FilterRule = React.createClass({
 		        };
 		    switch(rule.operator) {
 		    	case "equal":
+		    	case "isNull":
 		    		ajaxRetrieve("queryEqual.php");
 		    		break;
```

Comment:
isNull is equivalent to equal because you can search for a value equal to `null` if using direct access to CouchDB. All that needed to be changed for this was to specify that a value is `null` in the `queryEqual.php` script that the value should be `null` and not the string `"null"`. The same goes for `isNotNull`.